index.html

```diff
@@ -0,0 +1,77 @@
+<html>
+<body>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+<script src="versions.js"></script>
+
+<script>
+
+function addSeparator() {
+  $('#versionsList').append(
+    $('<li>')
+      .attr('role', 'separator')
+      .attr('class', 'divider')
+  )
+}
+
+function addVersion(version, versionName) {
+  versionName = versionName || version
+
+  $('#versionsList').append(
+    $('<li>').append(
+      $('<a>')
+        .attr('href', version + '/apidocs/index.html')
+        .attr('target', 'javadocs')
+        .text(versionName)
+    )
+  )
+}
+
+$(function() {
+
+  for (var ii = 0; ii < window.apiVersions.versions.length; ii++) {
+    let version = window.apiVersions.versions[ii];
+    var versionName = version;
+    if ( window.apiVersions.current === versionName) {
+      versionName = versionName + " [Current]";
+    }  
+    addVersion(version, versionName);
+  }
+
+  addSeparator();
+  // TODO: add dev version in the future
+  // addVersion('development', 'Development');
+  // addSeparator();
+
+  for (version of window.apiVersions.legacy) {
```

Comment:
As @robertdamphousse-okta mentioned, this doesn't work in IE. Follow the same logic as you did on line `34-39`

Reply:
doh